Export/Reports/RevenueExport.php

```diff
@@ -195,7 +195,9 @@ private function newEntry()
     private function getProcesses(): Traversable
     {
         $queryBuilder = $this->entityManager->getRepository(Process::class)->createQueryBuilder('p')
-            ->andWhere('p.originallyCompletedAt IS NOT NULL');
+            ->andWhere('p.originallyCompletedAt IS NOT NULL')
+            ->join('p.processClient', 'client')
+            ->andWhere('client.type = \'person\'');
```

Comment:
Should the `ProcessClientPerson::TYPE` constant be used here?

Reply:
Yes, of course.